pom.xml

```diff
@@ -12,7 +12,7 @@
   <version>1.36-SNAPSHOT</version>
 
   <properties>
-    <java.level>3</java.level>
+    <java.level>6</java.level>
```

Comment:
Did you check why this was so low? There was some effort spent (commented out generics), so there must be a reason, e.g. being able to show the "Java too old to run this" message properly?